content/en/docs/zero-code/java/spring-boot-starter/annotations.md

```diff
@@ -1,57 +1,80 @@
-span by annotating the method parameters with `@SpanAttribute`.
-
-> **Note**: this annotation can only be applied to bean methods managed by the
-> spring application context. To learn more about aspect weaving in spring, see
-> [spring-aop](https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#aop).
-
-| Feature     | Property                                   | Default Value | Description                       |
-| ----------- | ------------------------------------------ | ------------- | --------------------------------- |
-| `@WithSpan` | `otel.instrumentation.annotations.enabled` | true          | Enables the WithSpan annotations. |
+If you add the `WithSpan` annotation to a method, the method is wrapped in a
+span. The `SpanAttribute` annotation allows you to capture the method arguments
+as attributes.
 
 ```java
-import org.springframework.stereotype.Component;
-
 import io.opentelemetry.instrumentation.annotations.SpanAttribute;
 import io.opentelemetry.instrumentation.annotations.WithSpan;
-import io.opentelemetry.api.trace.Span;
-import io.opentelemetry.api.trace.SpanKind;
-
-/**
- * Test WithSpan
- */
-@Component
-public class TracedClass {
 
     @WithSpan
-    public void tracedMethod() {
+    public void tracedMethod(@SpanAttribute parameter) {
     }
+```
 
-    @WithSpan(value="span name")
-    public void tracedMethodWithName() {
-        Span currentSpan = Span.current();
-        currentSpan.addEvent("ADD EVENT TO tracedMethodWithName SPAN");
-        currentSpan.setAttribute("isTestAttribute", true);
-    }
+{{% alert title="Note" color="info" %}} The OpenTelemetry annotations use Spring
+AOP based on proxys.
+
+These annotations work only for the methods of the proxy. You can learn more in
+the
+[Spring documentation](https://docs.spring.io/spring-framework/reference/core/aop/proxying.html).
+
+In the following example, the `WithSpan` annotation won't do anything when the
+GET endpoint is called:
+
+```java
+@RestController
+public class MyControllerManagedBySpring {
 
-    @WithSpan(kind = SpanKind.CLIENT)
-    public void tracedClientSpan() {
+    @GetMapping("/ping")
+    public void aMethod() {
+        anotherMethod();
     }
 
-    public void tracedMethodWithAttribute(@SpanAttribute("attributeName") String parameter) {
+    @WithSpan
+    public void anotherMethod() {
     }
 }
 ```
+
+{{% /alert %}}
+
+{{% alert title="Note" color="info" %}}
+
+To be able to use the OpenTelemetry annotations, you have to add the Spring Boot
+Starter AOP dependency to your project: {{< tabpane text=true >}}
+{{% tab header="Maven (`pom.xml`)" lang=Maven %}} <dependencies> <dependency>
+<groupId>org.springframework.boot</groupId>
+<artifactId>spring-boot-starter-aop</artifactId> </dependency> </dependencies>
+{{% /tab %}} {{% tab header="Kotlin" lang=Kotlin %}} dependencies {
+implementation("org.springframework.boot:spring-boot-starter-aop") }
+{{% /tab %}} {{< /tabpane >}}
```

Comment:
i think the formatter might have messed this up a bit

this is what the [preview](https://deploy-preview-4806--opentelemetry.netlify.app/docs/zero-code/java/spring-boot-starter/annotations/) looks like currently:

<img width="1009" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/7630696/f8c306d9-58ad-4ceb-8c2e-783f65e3b707">

------

Github is making it difficult for me to post the suggested code change because of all the backticks, but this seemed to work for me:

Preview from local:

<img width="1006" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/7630696/b6346aee-6aeb-47fc-b632-7d9a59d1661c">

```plaintext
{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}

```
<dependencies>
  <dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-aop</artifactId>
  </dependency>
</dependencies>
```

{{% /tab %}} {{% tab header="Kotlin" lang=Kotlin %}}

```kotlin
dependencies {
  implementation("org.springframework.boot:spring-boot-starter-aop")
}
```

{{% /tab %}} {{< /tabpane >}}
```